data/blog/alex-gleason-receives-lts-grant.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "Long-Term Support For Alex Gleason"
+date: '2024-08-29'
+tags: ['OpenSats', 'grants', 'nostr', 'LTS']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/51-alex-gleason-receives-lts-grant.jpg']
+summary: "OpenSats Welcomes Alex Gleason as an LTS Grantee."
+---
+
+OpenSats is pleased to announce long-term support for [Alex
+Gleason](https://github.com/alexgleason), a prominent contributor to the nostr
+ecosystem.
+
+Alex is most recently known for launching [Ditto](https://soapbox.pub/ditto/), a
+decentralized, self-hosted social media server that prioritizes user control and
+community building, featuring a built-in nostr client and relay, compatible with
+Mastodon apps. He presented [Ditto at the Nostriga
+conference](https://www.youtube.com/watch?v=iZyqVGHjDGo) in August 2024.
+
+Beyond Ditto, Alex has made substantial contributions through several other key
+projects, including the [Mostr
+Bridge](https://soapbox.pub/blog/mostr-fediverse-nostr-bridge/), which connects
+nostr with the [Fediverse](https://activitypub.rocks/);
```

Comment:
The Fediverse link seems incorrect? @Arvin21M, unless you meant to link out to acitivitypub.rocks ?